ci/scripts/setup-integration-chaincode.sh

```diff
@@ -8,20 +8,21 @@
 # by the integration tool which doesn't have the local files. Removes the replace to stop the integration tests
 # breaking as the dot path doesn't exist there.
 
-DIR=$(pwd)
-CHAINCODE_DIR=$DIR/.azure-pipelines/resources/chaincode
-
-cd $CHAINCODE_DIR
+set -e -u -o pipefail
+ROOTDIR=$(cd "$(dirname "$0")" && pwd)
 
+CHAINCODE_DIR=$ROOTDIR/../../integrationtest/chaincode
+ls -lart $CHAINCODE_DIR
+pushd $CHAINCODE_DIR
 for testCC in */; do
-    cd $testCC
-    cat go.mod > tmp.go.mod
-    echo 'replace github.com/hyperledger/fabric-contract-api-go => ../../../..' >> go.mod
+    pushd $testCC
+    # cat go.mod > tmp.go.mod
```

Comment:
There's some commentated out code in this script which it would be good to tidy up